build/BUILD.wpt

```diff
@@ -0,0 +1,9 @@
+filegroup(
+  name = "url",
+  srcs = glob(
+    include = ["url/**/*"],
+    allow_empty = True,
+  ),
+  visibility = ["//visibility:public"],
+)
```

Comment:
We would need to export filegroups for each of the web platform test groups we want... we don't need to export filegroups for everything since most of the wpts will never be relevant